src/FacadeAccountCreation/FacadeAccountCreation.UnitTests/API/Controllers/EnrolmentsControllerTests.cs

```diff
@@ -98,7 +98,33 @@ public async Task When_service_successfully_deletes_user_then_return_NoContent(H
         response.Should().BeOfType<NoContentResult>();
         _mockMessagingService.Verify(x => x.SendRemovedUserNotification(It.IsAny<RemovedUserNotificationEmailModel>()), Times.Once);
     }
-    
+
```

Comment:
Did you not get complaints about the additional lines?

Reply:
I think this project doesn't have the stylecop rules set up so there are no warnings. Abid has removed this line.